### PR TITLE
Update snvphyl tools to 1.8.2 (latest)

### PR DESCRIPTION
**Pull request by Takadonet**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by Takadonet** [comment]
So looks like it finished building but fails on creating a container because of error with bcftools:

`bcftools: error while loading shared libraries: libcrypto.so.1.0.0: cannot open shared object file: No such file or directory`

Looks related to [issue](https://github.com/bioconda/bioconda-recipes/issues/12100) . Not sure how to approach to address this in my recipe.

File: recipes/bcftools-snvphyl-plugin/build.sh [hunk 1]
```diff
@@ -1,17 +1,41 @@
 #!/bin/sh
 
-#downloading outside of meta.yaml because bioconda lint cannot handle multiple list till they move to version 3 of bioconda
+export VERSION=1.9
+export C_INCLUDE_PATH=${PREFIX}/include
```
Comment (bgruening) [inline recipes/bcftools-snvphyl-plugin/build.sh]: Can this be removed now?

File: recipes/bcftools-snvphyl-plugin/build.sh [hunk 2]
```diff
@@ -1,17 +1,41 @@
...
 
-curl -L -s https://github.com/phac-nml/snvphyl-tools/archive/1.8.1.tar.gz > 1.8.1.tar.gz
-tar -zxvf 1.8.1.tar.gz
+CC=${CC}
+CXX=${CXX}
 
-#copy over custom bcftools plugin
-cp snvphyl*/bcfplugins/filter_snv_density.c plugins/
+ln -s ${CC} ${PREFIX}/bin/gcc
+ln -s ${CXX} ${PREFIX}/bin/g++
```
Comment (bgruening) [inline recipes/bcftools-snvphyl-plugin/build.sh]: g++ can probably removed as well now

**Comment by Takadonet** [comment]
@bgruening Good to go?

**Comment by bgruening** [comment]
Sure.

**Pull request merged**